.github/workflows/tests.yaml

```diff
@@ -135,18 +141,25 @@ jobs:
           firefox --version
           geckodriver --version
           xvfb-run --server-args="-screen 0, 1280x720x24" --auto-servernum \
-          geckodriver &> ./logs/geckodriver.log &
+          geckodriver --port=4444 &> ./logs/geckodriver.log &
+
+      - name: Start SafariDriver
+        if: ${{ !matrix.selenium-server && matrix.browser == 'safari' }}
+        run: |
+          defaults read /Applications/Safari.app/Contents/Info CFBundleShortVersionString
+          /usr/bin/safaridriver -p 4444 --diagnose &
```

Comment:
according to your code the driver port if 7055
did I misunderstand ?

Reply:
I'm going split this PR into two as @OndraM is suggesting, so it can be ignored.
But default port of Safari Driver is somewhat 7055 (nothing is said about it in official docs)